service/pipeline.ml

```diff
@@ -385,7 +383,11 @@ let test_pr ~ocluster ~master ~head =
     in
     Lint.check ~master ~packages src
   in
-  let builds = build_with_cluster ~ocluster ~analysis ~lint ~master commit_id in
+  let builds =
+    Node.root
+      (Node.leaf ~label:"(analysis)" (Node.action `Analysed latest_analysis)
+      :: build_with_cluster ~ocluster ~analysis ~lint ~master commit_id)
```

Comment:
Perhaps the `lint` leaf could similarly be extracted from `build_with_cluster`? (or `latest_analysis` could be passed to that function to preserve the pretty `Node.root [...]`)